homeassistant/components/media_player/mpd.py

```diff
@@ -270,12 +289,13 @@ def media_previous_track(self):
 
     def mute_volume(self, mute):
         """Mute. Emulated with set_volume_level."""
-        if mute is True:
-            self._muted_volume = self.volume_level
-            self.set_volume_level(0)
-        elif mute is False:
-            self.set_volume_level(self._muted_volume)
-        self._muted = mute
+        if 'volume' in self._status:
+            if mute is True:
+                self._muted_volume = self.volume_level
+                self.set_volume_level(0)
+            elif mute is False:
```

Comment:
```py
else:
```

Reply:
Fixed. I'm guessing the original author here thought it might be possible to get something other than a boolean as `mute`?